xarray/tests/test_cftimeindex_resample.py

```diff
@@ -149,6 +160,45 @@ def test_resample(freqs, closed, label, offset) -> None:
     )
 
 
+@pytest.mark.parametrize(
+    "freqs", FREQS_WITH_NON_TICK_RESAMPLE_FREQ, ids=lambda x: "{}->{}".format(*x)
+)
+@pytest.mark.parametrize("closed", [None, "left", "right"])
+@pytest.mark.parametrize("label", [None, "left", "right"])
+def test_resample_with_non_tick_resample_freq(freqs, closed, label) -> None:
```

Comment:
Since the `offset` and `origin` options are not relevant when resampling to a non-`Tick` frequency, I split these tests off into their function to avoid warnings and avoid unnecessary duplication of tests.